Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,43 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return EncryptOrDecrypt(message, shift, true);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            return EncryptOrDecrypt(message, shift, false);
+        }
+
+        /// <summary>
+        /// Encrypt or decrypt given message
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="shift"></param>
+        /// <param name="encrypt"> 
+        /// true to encrypt, false to decrypt
+        /// </param>
+        /// <returns></returns>
+        public static string EncryptOrDecrypt (string message, byte shift, bool encrypt)
+        {
+            if (message == null)
+            {
+                return null;
+            }
+
+            StringBuilder changedMessage = new StringBuilder();
+            for (int i = 0; i < message.Length; i++)
+            {
+                if (encrypt)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5028257)